docs/pages/enroll-resources/mcp-access/mcp-access.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: MCP Servers
+description: Guides to protecting MCP servers with Teleport
+---
+
+Teleport can provide secure connections to your MCP (Model Context Protocol)
+servers while improving both access control and visibility.
+
+![Architecture diagram for enrolling MCP servers with Teleport](../../../img/mcp-access/architecture-stdio.svg)
+
+Teleport protects MCP servers through the Teleport Application Service, which is
+a Teleport Agent service. For more information on agent services, read [Teleport
+Agent Architecture](../../reference/architecture/agents.mdx). You can also learn
+how to deploy a [pool of Teleport Agents](../agents/agents.mdx) to run multiple
+agent services.
```

Comment:
I think this paragraph will only confuse the user who just wants to get things working, I'd consider removing it.
```suggestion
```